intro-dplyr.Rmd

```diff
@@ -0,0 +1,343 @@
+---
+title: "Work with and manage your data efficiently in R: dplyr and tidyr"
+params:
+    datetime: "2018-10-26"
+    level: "Beginner" # or intermediate or advanced
+    videolink: "" # Keep empty if no link yet.
+    instructor: "Elena Dudukina"
+---
+
+```{r setup, echo=FALSE}
+source("R/utils.R")
+```
+
+## Session details
+
+- **Date of session**: `r format_date(params$datetime)`
+- **Instructor**: `r params$instructor`
+- **Session level**: `r params$level`
+    - **Part of the ["Beginner Series"](index.html)**
+- **Required packages to install**:
+
+    ```{r, eval=FALSE}
+    install.packages(c("dplyr", "tidyr"))
+    ```
+
+
+### Objectives
+
+1. To learn the difference between "messy" and "tidy" data and see the advantages of using data that are consistently structured.
+2. To get to know the tools that help tidying up and reshaping the data.
+3. To perform simple data transformations and create summaries.
+    
+**At the end of this session you will be able:**
+
+1. To perform simple data transformations using dplyr function `mutate()`.
+1. To select variables and observations to work with using dplyr functions `filter()`and `select()`.
+1. To order data by variable using dplyr function `arrange()`.
+1. To provide a simple data summary using dplyr functions `group_by()` and `summarise()`.
+1. To change the structure of the data using tidyr functions `gather()`, `separate()`, and `spread()`.
```

Comment:
This is a good list. I rearranged so the tidyr functions are on the bottom. I personally would suggest only focusing on gather and spread. Those are challenging concepts already.

Reply:
Also include using the pipe.